### PR TITLE
Review card

### PR DESCRIPTION
**Pull request by sashakaverina**
- added seed reviews (2 per each experience)
 - added review card (component) and in experience #show

### CODE REVIEW

**Comment by dmbf29** [comment]
can you give a screenshot for front-end components? 🙏

**Comment by sashakaverina** [comment]
> can you give a screenshot for front-end components? 🙏
Screen Shot 2022-01-08 at 22 10 57

File: app/views/experiences/show.html.erb [hunk 1]
```diff
@@ -17,7 +17,9 @@
   </div>
   <div class="row">
     <%# Map %>
-    <%#= render @experience.reviews %>
+    <% @experience.reviews.each do |review| %>
+      <%= render './reviews/review', review: review %>
+    <% end %>
```
Comment (trouni) [inline app/views/experiences/show.html.erb:22]: Actually, the commented line from before does exactly what the lines you've added do.
```suggestion
    <%= render @experience.reviews %>
```

File: app/views/reviews/_review.html.erb [hunk 2]
```diff
@@ -1 +1,14 @@
-<%# Review card for the experiences#show %>
+<div class="col col-6">
+  <div class="host-container p-3 mt-3">
+    <div class="host-name-container">
+      <%= cl_image_tag review.user.photo.key, class: "avatar-large" %>
```
Comment (trouni) [inline app/views/reviews/_review.html.erb:4]: I think this doesn't work on the latest main, but you can use the new avatar component that has been added:
```suggestion
      <%= render 'avatar', user: review.user, large: true %>
```

File: db/seeds.rb [hunk 3]
```diff
@@ -123,3 +123,17 @@
...
+  users.each do |user|
+    review = Review.new(
+      user: user,
+      content: Faker::Restaurant.review,
+      experience: experience,
+      rating: rand(1..5)
+    )
+ review.save!
+  end
+end
```
Comment (trouni) [inline db/seeds.rb:139]: PR #42 already added review seeds. Did you branch off the latest `main`?

Comment (sashakaverina) [reply db/seeds.rb]: pushed a new commit with edits:
 - deleted my seeds
 - changed avatar on reviews
 - changed render for reviews @trouni

**Review by trouni — APPROVED** [review]

**Pull request merged**